scripts/linters/pre_commit_linter.py

```diff
@@ -125,15 +125,16 @@
         _PARENT_DIR, 'oppia_tools', 'Pillow-%s' % common.PILLOW_VERSION),
     os.path.join(
         _PARENT_DIR, 'oppia_tools', 'psutil-%s' % common.PSUTIL_VERSION),
-    os.path.join('third_party', 'backports.functools_lru_cache-1.5'),
-    os.path.join('third_party', 'beautifulsoup4-4.7.1'),
-    os.path.join('third_party', 'bleach-3.1.0'),
+    os.path.join('third_party', 'backports.functools_lru_cache-1.6.1'),
+    os.path.join('third_party', 'beautifulsoup4-4.9.0'),
+    os.path.join('third_party', 'bleach-3.1.5'),
     os.path.join('third_party', 'callbacks-0.3.0'),
     os.path.join('third_party', 'gae-cloud-storage-1.9.22.1'),
     os.path.join('third_party', 'gae-mapreduce-1.9.22.0'),
     os.path.join('third_party', 'gae-pipeline-1.9.22.1'),
-    os.path.join('third_party', 'mutagen-1.42.0'),
-    os.path.join('third_party', 'soupsieve-1.9.1'),
+    os.path.join('third_party', 'packaging-20.3'),
```

Comment:
Add this in sorted order? (Also, why these things are defined in multiple places, can't we have this in one place?)

Reply:
Probably can, this is just a routine upgrade I will add an issue for this and think about ways how to resolve this.

Reply:
I have added #9251 to track this.